src/localization.ts

```diff
@@ -750,6 +750,26 @@ const LANG_TH: Localization = {
   text_fetch_error: "ไม่สามารถเชื่อมต่อได้"
 };
 
+// South Korean
+const LANG_KR: Localization = {
+  text_init: "초기화 중",
+
+  text_ready: "Anti-Robot 검증",
```

Comment:
I'm guessing there is no good translation for `Anti-Robot` in Korean?

Reply:
Very good question, for a personal need, I had translated by a person who speaks Korean, the translation is good I think :see_no_evil: